### PR TITLE
[DOC] Add first draft of macOS install guide, merge 2 doc dirs to sin…

### PR DESCRIPTION
**Pull request by marusinp**
…gle one.

Install guide was created running macOS 10.13 (High Sierra), different steps/errors for other versions can be added later in case of need.

### CODE REVIEW

File: docs/install_guide_macos.rst [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+**A:** Chyba pravdepodobne je na strane Homebrew, pri inštalácii Qt5 nevytvorí dve symlinky (ani po zavolaní brew link). Musíte ich vytvoriť ručne (ak treba, upravte verziu v ceste, návod bol robený pri nainštalovanej 5.9.2):
+
+::
+
+	ln -s /usr/local/Cellar/qt/5.9.2/mkspecs /usr/local/mkspecs
+	ln -s /usr/local/Cellar/qt/5.9.2/plugins /usr/local/plugins
+
+Ak príkaz zlyhá, pretože symlinky už existujú, zmažte ich a skúste znova.
+
+Q: Build error v 3dsoftviz/src/Mouse3d/LibMouse3d/Mac, riadok 68: 	Mouse3dMacDeviceGlob::ptr->OnMove( motionData );
```
Comment (kapecp) [inline docs/install_guide_macos.rst]: toto treba opravit - - malo by stacit
std::vector < float > motionData;
zamenit za
std::vector < double > motionData; 
(mozno const, resp. const_cast)

File: docs/install_guide_macos.rst [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+	/Library/Developer/CommandLineTools/usr/include/c++/v1/vector:542:5: note: candidate constructor not viable: no known conversion from 'std::vector<float>' to 'const std::__1::vector<double, std::__1::allocator<double> > &' for 1st argument
+	    vector(const vector& __x);
+	    ^
+	/Library/Developer/CommandLineTools/usr/include/c++/v1/vector:548:5: note: candidate constructor not viable: no known conversion from 'std::vector<float>' to 'std::__1::vector<double, std::__1::allocator<double> > &&' for 1st argument
+	    vector(vector&& __x)
+    ^
+
+**A:** Zakomentovať riadok (TEMPORARY!)
+
+Q: Runtime lua error ast.lua:9: module 'metrics.lookup' not found
```
Comment (kapecp) [inline docs/install_guide_macos.rst]: toto sa vyriesi novym luametric modulom, ked to bude na gitlab-e

File: docs/install_guide_macos.rst (hunk 1 above)
Comment (marusinp) [reply docs/install_guide_macos.rst]: dakujem pekne. bugfix pushujem do developu.

File: docs/install_guide_macos.rst (hunk 2 above)
Comment (marusinp) [reply docs/install_guide_macos.rst]: ok, odstranim z doc

**Pull request merged**